Sources/SwiftCurrent_SwiftUI/ViewInspector/InspectableSheet.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  InspectableSheet.swift
+//  SwiftCurrent_SwiftUI
+//
+//  Created by Morgan Zellers on 8/31/21.
+//  Copyright © 2021 WWT and Tyler Thompson. All rights reserved.
+//
+// swiftlint:disable file_types_order
+import SwiftUI
+
+@available(iOS 14.0, macOS 11, tvOS 14.0, watchOS 7.0, *)
+extension View {
+    func testableSheet<Sheet>(isPresented: Binding<Bool>, onDismiss: (() -> Void)? = nil, @ViewBuilder content: @escaping () -> Sheet
```

Comment:
I get why we do this.  And I realize the name is pretty self-explanatory, but I wonder if we should still add a little more documentation to this?  I'm conflicted, so feel free to say no.

Reply:
No.

Reply:
To expand on that: This isn't a public symbol, nor is it one that you have to worry about even if you're a consumer using ViewInspector doing an `@testable import SwiftCurrent_SwiftUI`. So unless it's somehow unclear to maintaners I think we're okay.